cpplib/Tests/MainTest.cpp

```
@@ -93,6 +97,309 @@ struct FMIC_TS {
+//	auto buildresult = ucb.build(xyz, types);
+//
+//	geometry::SpatialGrid<FloatingPointType> space;
+//	space.build(buildresult.atoms.points, cell, 6.0);
+//	auto bonds = WITH_LOG_M(space, get_bonds, false);
+//
+//	std::vector<bool> bools(buildresult.atoms.points.size(), true);
+//	Diagram diag(buildresult.atoms.points, bonds, cell, bools);
+//	auto ce = diag.extractCells();
+//
+//	cpplib::voronoi::VoronoiFused vf(ce, cell.fracToCart());
+//
```

Cppcheck (2.21.0)

[performance] Function parameter 'point' should be passed by const reference.

(passedByValue)